Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml

```
@@ -344,3 +346,29 @@
+
+- type: entity
+  parent: ClothingNeckBase
+  id: OmniCloak
+  name: Omni-man cloak
+  prefix: ADMEME
+  description: You fell strenght when you wearing this.  #Goobstation
+  components:
+  - type: Sprite
+    sprite: Clothing/Neck/Cloaks/goliathcloak.rsi
+  - type: ToggleableClothing # Goobstation - Modsuits change
+    clothingPrototypes: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)